opensuse_buildservice/notqmail.spec

```diff
@@ -0,0 +1,423 @@
+%attr(0755,root,qmail)            %{qmaildir}/bin/sendmail
+%attr(0755,root,qmail)            %{qmaildir}/bin/except
+%attr(0755,root,qmail)            %{qmaildir}/bin/qmail-pop3d
+%attr(0711,root,qmail)            %{qmaildir}/bin/qmail-clean
+%attr(0755,root,qmail)            %{qmaildir}/bin/qbiff
+%attr(0755,root,qmail)            %{qmaildir}/bin/qmail-qstat
+%attr(0755,root,qmail)            %{qmaildir}/bin/mailsubj
+%attr(0755,root,qmail)            %{qmaildir}/bin/qmail-tcpok
+%attr(0755,root,qmail)            %{qmaildir}/bin/qmail-tcpto
+%attr(0755,root,qmail)            %{qmaildir}/bin/qmail-qmtpd
+%attr(0700,root,qmail)            %{qmaildir}/bin/qmail-start
+%attr(4711,qmailq,qmail)          %{qmaildir}/bin/qmail-queue
+%attr(0755,root,qmail)            %{qmaildir}/bin/maildirmake
+%attr(0755,root,qmail)            %{qmaildir}/bin/qmail-qmqpd
+%attr(0711,root,qmail)            %{qmaildir}/bin/qmail-pw2u
+%attr(0755,root,qmail)            %{qmaildir}/bin/datemail
+%attr(0700,root,qmail)            %{qmaildir}/bin/qmail-newmrh
+%attr(0711,root,qmail)            %{qmaildir}/bin/qmail-local
+%attr(0755,root,qmail)            %{qmaildir}/bin/predate
+%attr(0711,root,qmail)            %{qmaildir}/bin/qmail-send
+%attr(0755,root,qmail)            %{qmaildir}/bin/forward
+%attr(0711,root,qmail)            %{qmaildir}/bin/qmail-rspawn
+%attr(0755,root,qmail)            %{qmaildir}/bin/qmail-qmqpc
+%attr(0755,root,qmail)            %{qmaildir}/bin/condredirect
+%attr(0755,root,qmail)            %{qmaildir}/bin/tcp-env
+%attr(0711,root,qmail)            %{qmaildir}/bin/qmail-remote
+
+%attr(0755,root,qmail)            %{qmaildir}/boot/proc+df
+%attr(0755,root,qmail)            %{qmaildir}/boot/proc
+%attr(0755,root,qmail)            %{qmaildir}/boot/home
+%attr(0755,root,qmail)            %{qmaildir}/boot/binm2+df
+%attr(0755,root,qmail)            %{qmaildir}/boot/binm3
+%attr(0755,root,qmail)            %{qmaildir}/boot/home+df
+%attr(0755,root,qmail)            %{qmaildir}/boot/binm1
+%attr(0755,root,qmail)            %{qmaildir}/boot/binm2
+%attr(0755,root,qmail)            %{qmaildir}/boot/binm1+df
+%attr(0755,root,qmail)            %{qmaildir}/boot/binm3+df
+%if "%{?_unitdir}" != ""
+                                  %{_unitdir}/%{name}.service
+%endif
+
+%if %noperms == 0
+%if 0%{?suse_version} >= 1120
+%attr(644,root,root)                               %{_sysconfdir}/permissions.d/%{name}-permissions
+%attr(644,root,root)                               %{_sysconfdir}/permissions.d/%{name}-permissions.secure
+%endif
+%endif
+%if %noperms == 0
+%if 0%{?suse_version} >= 1120
+%verify (not user group mode) %attr(6551, qmailq, qmail)   %{qmaildir}/bin/qmail-queue
+%verify (not user group mode) %attr(2555, alias, qmail)    %{qmaildir}/alias
+%endif
+%endif
+
+%files doc
+%{_mandir}/man?/*%{?ext_man}
+%{nq_docdir}
+
+### SCRIPTLETS ###############################################################################
+%verifyscript
+ID=$(id -u)
```

Comment:
Why do we need this check?

Reply:
I need to check this. Will get back on this. In my project I call instcheck to check and fix permissions.